### PR TITLE
Changing all the references from `Flash` to `OpenFL`.

### PR DESCRIPTION
**Pull request by MAJigsaw77**
The title tells everything this commit do.

### CODE REVIEW

**Comment by Geokureli** [comment]
ive been putting this off because it will likely cause merge conflicts with almost every other active PR, but let's try it. gimme some time, first

File: flixel/input/mouse/FlxMouse.hx [hunk 1]
```diff
@@ -17,11 +17,11 @@ import flixel.system.FlxAssets;
...
-import flash.Vector;
-import flash.geom.Matrix;
-import flash.geom.Point;
-import flash.ui.MouseCursor;
-import flash.ui.MouseCursorData;
+import openfl.Vector;
+import openfl.geom.Matrix;
+import openfl.geom.Point;
+import openfl.ui.MouseCursor;
+import openfl.ui.MouseCursorData;
```
Comment (Geokureli) [inline flixel/input/mouse/FlxMouse.hx:24]: looks like flash is failing here because there is no `openfl.ui.MouseCursorData`

Comment (Geokureli) [reply flixel/input/mouse/FlxMouse.hx:24]: https://github.com/HaxeFlixel/flixel/blob/dev/flixel/system/macros/FlxDefines.hx#L144
I think FLX_NATIVE_CURSOR is only used with flash, so lets just use `flash.ui.MouseCursorData`

Comment (MAJigsaw77) [reply flixel/input/mouse/FlxMouse.hx:24]: ok?

Comment (MAJigsaw77) [reply flixel/input/mouse/FlxMouse.hx]: Nah

**Comment by Geokureli** [comment]
Thanks, hopefully this fixes most macro issues

**Pull request merged**